facebook-commerce.php

```diff
@@ -696,6 +696,23 @@ public function get_product_count() {
 	public function allow_full_batch_api_sync() {
 		$default_allow_sync = true;
```

Comment:
It would be nicer to move this to the code block with the deprecated code, since it's not used at all with the new filter.